### DISCUSSION TITLE
More Pipeline Dropdowns

### DISCUSSION THREAD

**Discussion by TCROC** [GodotECS/godex#231] [Ideas & Proposals]
I really like the Pipeline drop down.  It makes it very easy to select a pipeline without creating typos:

![image](https://user-images.githubusercontent.com/24307049/116273884-24b50f00-a750-11eb-8ac1-957af56d283f.png)

It would be nice to be able to do that in other parts of the project as well.  Unless there is a design choice for having to enter these pipelines manually?

![image](https://user-images.githubusercontent.com/24307049/116274045-4a421880-a750-11eb-92bf-d34a7394433a.png)

If there is a design reason, maybe there can be a drop-down arrow inside the textbox that can be selected in order to present the user with a list of options that they can select to prefill the textbox?

**Comment by AndreaCatania** [comment]
Yup, this is doable. I initially used the input line, since faster to do. In a later pass I added the drop down. Add it there too is definitely a good idea and it's definitely doable.

**Comment by AndreaCatania** [comment]
Moving the proposal to discussions to keep the issue just for things to fix.
